linkedList.js

```diff
@@ -4,5 +4,116 @@
  * @return {Object} an object exposing methods to be used to manipulate a linked list
  */
 function linkedListGenerator(){
+  var head = null;
+  var tail = null;
+  var length = 0;
+  // var node = {
+  //           value: null,
+  //           next: {},
+  //         };
+  var linkedList = {
+    getHead: function(){
+              return head;
+            },
+    getTail: function(){
```

Comment:
No real problems here, but consider using the `tail` variable that you've declared above. Just like how you've leveraged the `head` variable in the `getHead` method.